### PR TITLE
Add note for ECR private registry on org policies

### PR DESCRIPTION
**Pull request by mustard-mh**
## Related Issue(s)

Fixes https://github.com/gitpod-io/website/pull/3986#issuecomment-1737152066

## How to test

Visit  https://deploy-preview-3992--gitpod-kumquat.netlify.app//docs/configure/orgs/policies#default-workspace-image

## Release QA Checklist ✔️

<details>
  <summary>QA Checklist </summary>

## [Priority: High] Functional Testing

-   [x] Test all links to make sure they are working correctly
-   [x] Verify that all forms (sales, support, docs feedback) are functioning properly and data is being captured correctly
-   [x] Check all buttons and calls-to-action to ensure they are leading to the correct pages
-   [x] Ensure that the website is mobile responsive and functions appropriately on different devices
-   [x] Please make sure that page is best for Light/Dark Mode. ⇒ Text/Media Changes

## [Priority: High] Content Testing

-   [x] Check all website copies for accuracy and spelling/grammar errors
-   [x] Verify that all images and videos are displaying properly
-   [x] Ensure that all downloadable files are working correctly
-   [x] Check that all external links are functioning properly

## [Priority: Medium] Performance Testing

-   [x] Try to use optimal media assets. ([Decision Record - How To](https://github.com/gitpod-io/website/blob/main/docs/adr/0003-optimal-assets-format.md))
-   [x] Test website load times to ensure that the website is loading quickly
-   [x] Verify that the website can handle high traffic volumes without crashing
-   [x] Check that all website functionality is working properly under heavy load
-   [x] Perform cross-browser testing to ensure that the website works properly on all major browsers (Google Chrome, Firefox, Edge)

## [Priority: Medium] Accessibility Testing

-   [x] Ensure that the website meets basic accessibility guidelines.
-   [x] Test that users with disabilities can easily navigate and use the website
-   [x] Verify that all website content can be accessed using assistive technologies

By including these key items in your website release QA checklist, you can ensure that your website is fully functional, user-friendly, and meets your business requirements.

## Security Testing

> **Note**:💡 If we are changing something related to cookies, data storing, or analytics.

-   [x] Verify that the website is HTTPS secure and that there are no security vulnerabilities
-   [x] Ensure that all sensitive data is being stored and transmitted securely

</details>

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/3992"></a>

### CODE REVIEW

File: src/content/docs/configure/orgs/policies.md [hunk 1]
```diff
@@ -32,4 +32,6 @@ Workspace sharing is enabled by default. Through this policy, Organization Owner
 
 > **Note:** The user must have the **Organization Owner** role to update the Organization default workspace image. All Organization members can view this setting only.
 
+> **Note:** Authentication for the images is based on what the Gitpod installation can access. For **Gitpod Cloud users**, only public images are currently supported. For **Gitpod Dedicated**, the authentication is inherited from the installation. For instance, if you are using [ECR private registry](/docs/gitpod-dedicated/guides/use-private-ecr-repos-for-workspace-images#using-private-ecr-repositories-for-workspace-images) support that will work with the Organization setting.
```
Comment (loujaybee) [inline src/content/docs/configure/orgs/policies.md:35]: Let's not overuse the callouts, they can be quite visually jarring. Let's add a sub section. 

```suggestion

### Workspace Image Authentication

Authentication for the images is based on what the Gitpod installation can access. For **Gitpod Cloud users**, only public images are currently supported. For **Gitpod Dedicated**, the authentication is inherited from the installation. For instance, if you are using [ECR private registry](/docs/gitpod-dedicated/guides/use-private-ecr-repos-for-workspace-images#using-private-ecr-repositories-for-workspace-images) support that will work with the Organization setting.
```

Comment (loujaybee) [inline src/content/docs/configure/orgs/policies.md]: We should be sparing with bolding—it's a tool for drawing attention and/or creating formatting consistency. 

Is there a good reason for us to bold here?

```suggestion
> **Note:** Authentication for the images is based on what the Gitpod installation can access. For Gitpod Cloud users, only public images are currently supported. For Gitpod Dedicated, the authentication is inherited from the installation. For instance, if you are using [ECR private registry](/docs/gitpod-dedicated/guides/use-private-ecr-repos-for-workspace-images#using-private-ecr-repositories-for-workspace-images) support that will work with the Organization setting.
```

File: src/content/docs/configure/orgs/policies.md [hunk 2]
```diff
@@ -32,4 +32,9 @@ Workspace sharing is enabled by default. Through this policy, Organization Owner
 
 > **Note:** The user must have the **Organization Owner** role to update the Organization default workspace image. All Organization members can view this setting only.
 
+
+### Workspace Image Authentication
+
+Authentication for the images is based on what the Gitpod installation can access. For **Gitpod Cloud users**, only public images are currently supported. For **Gitpod Dedicated**, the authentication is inherited from the installation. For instance, if you are using [ECR private registry](/docs/gitpod-dedicated/guides/use-private-ecr-repos-for-workspace-images#using-private-ecr-repositories-for-workspace-images) support that will work with the Organization setting.
+
```
Comment (loujaybee) [inline src/content/docs/configure/orgs/policies.md:39]: We should be sparing with bolding—it's a tool for drawing attention and/or creating formatting consistency. 

Is there a good reason for us to bold here?

```suggestion

### Workspace Image Authentication

Authentication for the images is based on what the Gitpod installation can access. For Gitpod Cloud users, only public images are currently supported. For Gitpod Dedicated, the authentication is inherited from the installation. For instance, if you are using [ECR private registry](/docs/gitpod-dedicated/guides/use-private-ecr-repos-for-workspace-images#using-private-ecr-repositories-for-workspace-images) support that will work with the Organization setting.

```

File: src/content/docs/configure/orgs/policies.md (hunk 1 above)
Comment (mustard-mh) [reply src/content/docs/configure/orgs/policies.md]: @loujaybee  Nope, I just want to split them to make it easier to find what you need. Will a list be helpful?

**Review by loujaybee — APPROVED** [review]
🙏 - Thanks @mustard-mh

**Comment by mustard-mh** [comment]
Going to move Workspace Image doc above

image

**Pull request merged**